jdbc/src/test/java/org/apache/zeppelin/jdbc/SqlCompleterTest.java

```diff
@@ -0,0 +1,313 @@
+package org.apache.zeppelin.jdbc;
```

Comment:
@sotnich All .java files should have ASF license header on top of each file. You didn't add it and you can add it by copying the header from one of other java files. :-)